pkgs/os-specific/linux/drbd/driver.nix

```diff
@@ -42,8 +42,9 @@ stdenv.mkDerivation rec {
   '';
 
   postFixup = ''
-    for ko in $out/lib/modules/${kernel.modDirVersion}/kernel/drivers/block/drbd/*.ko; do
+    for ko in $out/lib/modules/${kernel.modDirVersion}/kernel/drivers/block/drbd9/*.ko; do
       xz --compress -6 --threads=0 $ko
+      chmod 0444 $ko.xz
```

Comment:
I wish there was a comment outlining the reason for this permission fixup.

Reply:
Hi Astro, i have updated the PR description.